prime-dotnet-webapi/Migrations/20211113005943_AddedPaperSubmissionTemplate.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class AddedPaperSubmissionTemplate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropIndex(
+                name: "IX_PharmanetTransactionLog_TransactionId",
+                table: "PharmanetTransactionLog");
```

Comment:
You will need to re make this migration, this dropping and re-adding of the index on this table was do to a mistake in another PR.